streams-tests/jvm/src/test/scala/zio/stream/StreamSpec.scala

```diff
@@ -1736,31 +1755,12 @@ class StreamSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRu
       .runCollect must_=== List()
   }
 
-  private def throttleEnforceShortCircuits = {
```

Comment:
Good that these are deleted.

Reply:
Ah I had to get rid of something because of the `UTF8 string too long` error.

Reply:
Finally, I thought it was just me and I was going crazy.